codalab/lib/bundle_cli.py

```diff
@@ -1054,6 +1054,11 @@ def do_store_command(self, args):
                 "url": args.url,
                 "authentication": args.authentication,
             }
+            if args.storage_type is None:
+                if args.url.startswith("azfs://"):
+                    bundle_store_info["storage_type"] = "azure_blob"
```

Comment:
Also, let's actually move this logic to here (https://github.com/codalab/codalab-worksheets/blob/master/codalab/rest/bundles.py#L514-L522) to be consistent with how storage_format is inferred.